src/infrastructure/jira/jira-service.ts

```diff
@@ -180,6 +176,32 @@ class JiraService {
 		);
 	};
 
+	updateIngestedDesignsIssueProperty = async (
+		issueIdOrKey: string,
+		{ url }: AtlassianDesign,
+		connectInstallation: ConnectInstallation,
+	): Promise<void> => {
+		const storedValue =
+			await this.getStoredIssuePropertyValue<IngestedDesignUrlIssuePropertyValue>(
+				issueIdOrKey,
+				propertyKeys.INGESTED_DESIGN_URLS,
+				connectInstallation,
+				INGESTED_DESIGN_URL_VALUE_SCHEMA,
+			);
+		if (storedValue && storedValue.some((value) => value === url)) {
```

Comment:
**nitpick:** Consider using the `?` operator to simplify the expression:

```ts
if (storedValue?.some((value) => value === url)) {
  // ...
}
```

The same solution could be applied to the similar code in `updateAttachedDesignUrlV2IssueProperty`.